test/views/page_view_test.exs

```diff
@@ -1,3 +1,86 @@
 defmodule Askbywho.PageViewTest do
   use Askbywho.ConnCase, async: true
+  alias Askbywho.PageView
+  alias Askbywho.Repo
+  alias Askbywho.Email
+  alias Askbywho.Brand
+
+  test "#count_emails should get the number of emails on database" do
+    %Email{}
+      |> Email.changeset(%{"email" => "test@email.com", "name" => "Test", "brands" => "Adidas, Nike"})
```

Comment:
Should have "DRY" here, and consider creating a seed function that would just pre-popular the database with the right base set of emails.